javascript/javascript1/week2/get-students.js

```diff
@@ -0,0 +1,38 @@
+const class07Students = [];
+function addStudentToClass(studentName){
+    if(class07Students.length ===6 && studentName!=="Queen")
+       { return console.log(`Can not  add  more students to class 07`);
+    }
+    else if(class07Students.includes(studentName)){
+       return console.log(`Student ${studentName} is already in the class`); 
+    }
+    else if(studentName==="Queen"){
+    class07Students.push(studentName)
+   return console.log(`${studentName} can be added to class 07`);
+}
+    
+
+    else if(studentName===" "){
```

Comment:
`" "` is technically an empty string (which only contains a space character) but so is `""`. I would just check for `""` and leave things like `" "` (and even more spaces) for another time when unless you want to play with string trimming. You can use Google to search for "javascript trim string" if you want to know more.